packages/editor/src/client/atoms/chooseEditImage.tsx

```diff
@@ -22,20 +22,22 @@ export function ChooseEditImage({
 }: ChooseEditImageProps): JSX.Element {
   const {t} = useTranslation()
   header = header ?? t('chooseEditImage.header')
+  const panelHeight = image ? (image.height < 240 ? image.height : 240) : 240
   return (
-    <Panel
-      header={header}
-      style={{
-        height: 300
-      }}>
+    <Panel header={header} style={{}}>
```

Comment:
why the empty object in style? if you don't have anything you can remove the property.

Reply:
Fixed it